README.md

```diff
@@ -1,6 +1,15 @@
 Sample Python SP
 ================
 
+<aside class="warning">
+<br>This sample SP has been retired.  It was used for early prototyping for integrations with login.gov and has not been maintained. It has confirmed vulnerabilities and should not be used for production itegrations.</br> 
+<br>For maintained examples of SAML integrations with login.gov please refer to:
+<ul>https://github.com/18F/identity-saml-rails</ul>
+<ul>https://github.com/18F/identity-saml-sinatra</ul>
+</br>
```

Comment:
This is a bit nitpicky, but since this is markdown and not HTML, it would be more natural to write markdown syntax instead.

Those `<ul>` entries were presumably meant to be `<ul><li>...`. But in markdown that could just be `* https://...`.